examples/web/src/misc/UseSignFrameAction.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  FrameVerifySignature,
+  FrameVerifySignatureResult,
+  LensClient,
+  development,
+} from '@lens-protocol/client';
+import {
+  FramesEip721TypedDataSpec,
+  Profile,
+  publicationId,
+  uri,
+  useIdentityToken,
+  useSignFrameAction,
+  useStorage,
+} from '@lens-protocol/react-web';
+import { useEffect, useMemo, useState } from 'react';
+import { toast } from 'react-hot-toast';
+
+import { RequireProfileSession } from '../components/auth';
+
+function ServerVerifySignature({ toVerify }: { toVerify: FrameVerifySignature }) {
+  const [verified, setVerified] = useState<FrameVerifySignatureResult>();
+  const storage = useStorage();
+  const client = useMemo(
+    () =>
+      new LensClient({
+        environment: development,
+        storage,
+      }),
+    [storage],
+  );
+
+  useEffect(() => {
+    void (async () => {
+      const status = await client.frames.verifyFrameSignature(toVerify);
+      setVerified(status);
+    })();
+  }, [client, toVerify]);
+
+  return <div>Server verifications status: {verified}</div>;
+}
+
+export function UseSignFrameActionInner({ profile }: { profile: Profile }) {
+  const [frameActionToVerify, setFrameActionToVerify] = useState<FrameVerifySignature>();
+
+  const { data, execute, loading } = useSignFrameAction();
+  const identityToken = useIdentityToken();
+
+  const sign = async () => {
+    const result = await execute({
+      actionResponse: '0x0000000000000000000000000000000000000000',
+      buttonIndex: 2,
+      inputText: 'Hello, World!',
+      profileId: profile.id,
+      pubId: publicationId('0x01-0x01'),
+      specVersion: FramesEip721TypedDataSpec.OnePointOnePointOne,
+      state: '{"counter":1,"idempotency_key":"431b8b38-eb4d-455b"}',
+      url: uri('https://mylensframe.xyz'),
+    });
+
+    if (result.isFailure()) {
+      toast.error(result.error.message);
+      return;
+    }
+
+    const { signature, signedTypedData } = result.value;
+
+    if (!identityToken) {
+      throw new Error('No identity token found');
+    }
+
+    setFrameActionToVerify({
+      signature,
+      signedTypedData: {
+        domain: signedTypedData.domain,
+        types: signedTypedData.types,
+        value: signedTypedData.message, // TODO: this mapping is annoying
```

Comment:
@cesarenaldi in the react sdk we alias typed data `value` as `message` which is now "disturbing" with frame action verification, is there any meaning behind it? can we remove the alias from all typed data? it has impact on how the signer sees them.